webplayer-example-vanilla-js/server.js

```diff
@@ -28,12 +28,12 @@ app.post('/api/auth', jsonParser, async (req, res) => {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
-        'Authorization': req.body.token,
+        Authorization: req.body.token,
```

Comment:
The other auth token has `// your API token` written above it, may want that here too if its useful as a note